lib/cc/presenters/github_pull_requests_presenter.rb

```diff
@@ -13,7 +13,7 @@ def initialize(payload, repo_config)
       end
 
       def success_message
-        if @repo_config.pr_status_quality_stats?
+        if issue_counts_in_payload?
```

Comment:
Yeah. If we are the one controlling the message, I think this conditional is unnecessary. YAGNI.